### ISSUE TITLE
JUnit reporter: Invalid content was found starting with element 'error'

### ISSUE THREAD

**Issue by nblock** [postmanlabs/newman#1678]
1. Newman Version (can be found via `newman -v`): 4.0.2
 2. OS details (type, version, and architecture): Jenkins 2.212.2, xUnit 2.1.0
 3. Are you using Newman as a library, or via the CLI: CLI
 3. Did you encounter this recently, or has this bug always been there: Just started using it
 4. Expected behaviour: xUnit should be able to parse the XML file.
 5. Command / script used to run Newman: `newman run ... --reporters cli,junit --reporter-junit-export $WORKSPACE/postman.xml`

Error message from xUnit during parsing:
```
WARNING: The file '/home/jenkins/workspace/jobname/postman.xml' is an invalid file.
WARNING: At line 4 of file:/home/jenkins/workspace/jobname/postman.xml:cvc-complex-type.2.4.a: Invalid content was found starting with element 'error'. One of '{testsuite, properties, testcase, system-out, system-err}' is expected.
```

The XML file generated by running newman:
```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Call all login pages" tests="109" time="38.696">
  <testsuite name="open login page" id="7f72e270-d492-452e-a41e-d1f27a2d5de9" tests="4" failures="5" errors="2" time="0.355">
    <error>
      <![CDATA[Iteration: 9
RequestError: Error: write EPROTO 140111312403328:error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:../deps/openssl/openssl/ssl/s23_clnt.c:827:

    at WriteWrap.afterWrite [as oncomplete] (net.js:844:14)

---
testScriptError: Cannot read property 'has' of undefined
---
]]>
    </error>
    <testcase name="Body does NOT contain string maintenance mode" time="0.337"/>
    <testcase name="Status code is 403" time="0.337"/>
    <testcase name="Response time is less than 2500ms" time="0.337"/>
    <testcase name="Body contains string user login" time="0.337"/>
  </testsuite>
</testsuites>
```

I guess the text inside the `error` node should be wrapped inside `system-err`

**Comment by carlowahlstedt** [comment]
Looks like someone found [this issue](https://github.com/carlowahlstedt/NewmanPostman_VSTS_Task/issues/42#event-1992211269) using Newman via AzureDevOps.

**Comment by donotello** [comment]
Are you using `postman.sendRequest` in your tests? If so then it's related to  #1303.

**Comment by codenirvana** [comment]
This is fixed in Newman v4.4.1.
